modules/post/windows/gather/enum_application_artifacts_packrat.rb

```diff
@@ -0,0 +1,918 @@
+    {
+      :application=> "opera",
+      :category => "browsers",
+      :filetypes => "web_history",
+      :path => 'AppData',
+      :dir => "Opera Software",
+      :artifact=> "Visited Links",
+      :description => "Opera Visited Links"},
+    ## SRware
+    {
+      :application=> "srware",
+      :category => "browsers",
+      :filetypes => "logins",
+      :path => 'LocalAppData',
+      :dir => "Chromium",
+      :artifact=> "Login Data",
+      :description => "SRware's saved Username & Passwords"},
+    {
+      :application=> "srware",
+      :category => "browsers",
+      :filetypes => "web_history",
+      :path => 'LocalAppData',
+      :dir => "Chromium",
+      :artifact=> "Cookies",
+      :description => "SRware's Cookies"},
+    {
+      :application=> "srware",
+      :category => "browsers",
+      :filetypes => "web_history",
+      :path => 'LocalAppData',
+      :dir => "Chromium",
+      :artifact=> "History",
+      :description => "SRware's Visited websites history"},
+    ## Safari
+    {
+      :application=> "safari",
+      :category => "browsers",
+      :filetypes => "logins",
+      :path => 'AppData',
+      :dir => "Apple Computer",
+      :artifact=> "keychain.plist",
+      :description => "Safari History"},
+    ## SeaMonkey
+    {
+      :application=> "seamonkey",
+      :category => "browsers",
+      :filetypes => "logins",
+      :path => 'AppData',
+      :dir => "Mozilla",
+      :artifact=> "logins.json",
+      :description => "SeaMonkey's saved Username & Passwords"},
+    {
+      :application=> "seamonkey",
+      :category => "browsers",
+      :filetypes => "logins",
+      :path => 'AppData',
+      :dir => "Mozilla",
+      :artifact=> "cert8.db",
+      :description => "SeaMonkey's saved Username & Passwords"},
+    {
+      :application=> "seamonkey",
+      :category => "browsers",
+      :filetypes => "logins",
+      :path => 'AppData',
+      :dir => "Mozilla",
+      :artifact=> "key3.db",
+      :description => "SeaMonkey's saved Username & Passwords"},
+    {
+      :application=> "seamonkey",
+      :category => "browsers",
+      :filetypes => "web_history",
+      :path => 'AppData',
+      :dir =>'Mozilla',
+      :artifact=> "formhistory.sqlite",
+      :description => "SeaMonkey's saved Username"},
+    {
+      :application=> "seamonkey",
+      :category => "browsers",
+      :filetypes => "web_history",
+      :path => 'AppData',
+      :dir => "Mozilla",
+      :artifact=> "places.sqlite",
+      :description => "SeaMonkey History"},
+    {
+      :application=> "seamonkey",
+      :category => "browsers",
+      :filetypes => "cookies",
+      :path => 'AppData',
+      :dir => "Mozilla",
+      :artifact=> "cookies.sqlite",
+      :description => "SeaMonkey's cookies"}
+  ]
+  @@success_count = 0
+  @@try_count = 0
+
+  def run
+    print_line("\nPackRat is searching and gathering...\n")
+    print_line("Filtering based on these selections: \n")
```

Comment:
@jvazquez-r7 I think instead of print_line, it should be print_status. And both the initial and trailing \n won't be needed for print_status.